src/OrchardCore.Modules/OrchardCore.Workflows/Assets/Scripts/workflow-editor.ts

```diff
@@ -145,26 +151,36 @@ class WorkflowEditor extends WorkflowCanvas {
             activityElements.each((_, item) => {
                 var activityElement = $(item);
                 activityElement.popover({
-                    trigger: 'manual',
+                    trigger: "manual",
                     html: true,
                     content: () => {
-                        const $content: JQuery<Element> = activityElement.find('.activity-commands').clone();
-                        const startButton = $content.find('.activity-start-action');
-                        const isStart = activityElement.data('activity-start') === true;
-                        const activityId: number = activityElement.data('activity-id');
-                        startButton.attr('aria-pressed', activityElement.data('activity-start'));
-                        startButton.toggleClass('active', isStart);
-
-                        $content.on('click', '.activity-start-action', e => {
+                        const $content: JQuery<Element> = activityElement.find(".activity-commands").clone();
+                        const startButton = $content.find(".activity-start-action");
+                        const isStart = activityElement.data("activity-start") === true;
+                        const activityId: number = activityElement.data("activity-id");
+                        startButton.attr("aria-pressed", activityElement.data("activity-start"));
+                        startButton.toggleClass("active", isStart);
+
+                        $content.on("click", ".activity-start-action", (e) => {
                             e.preventDefault();
                             const button = $(e.currentTarget);
 
-                            const isStart = button.is('.active');
-                            activityElement.data('activity-start', isStart);
-                            activityElement.toggleClass('activity-start', isStart);
+                            let isStart = button.is(".active");
+
+                            if (isStart) {
+                                button.removeClass("active");
+                                isStart = false;
+                            }
+                            else {
+                                button.addClass("active");
+                                isStart = true;
+                            }
```

Comment:
This is just a workaround of the actual problem, being that bootstrap is included twice. The button is set up as a bootstrap toggle button, but is now toggled twice all the time.

Reply:
I did not see it toggled twice when debugging.

Reply:
I've spend way too much time on this, it's toggling twice - believe me 😇

The issue can be fixed by removing the bootstrap dependencies here: https://github.com/OrchardCMS/OrchardCore/blob/cb2e458b771b3399943720ca85955f54a14c808e/src/OrchardCore.Modules/OrchardCore.Workflows/ResourceManagementOptionsConfiguration.cs#L50 and https://github.com/OrchardCMS/OrchardCore/blob/cb2e458b771b3399943720ca85955f54a14c808e/src/OrchardCore.Modules/OrchardCore.Workflows/ResourceManagementOptionsConfiguration.cs#L59